modules/bootkube/resources/manifests/machine-api-operator.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: machine-api-operator
+  namespace: kube-system
+  labels:
+    k8s-app: machine-api-operator
+    managed-by-channel-operator: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: machine-api-operator
+  template:
+    metadata:
+      labels:
+        k8s-app: machine-api-operator
+        tectonic-app-version-name: machine-api
+    spec:
+      containers:
+      - name: machine-api-operator
+        image: quay.io/alberto_lamela/machine-api-operator:a661677 # TODO: move this to openshift org
+        command:
+        - "/machine-api-operator"
```

Comment:
Is the rule to put all the binaries under `/`? I noticed his pattern is used on multiple places (even in cluster-api upstream repo). IMHO, whenever possible I prefer `/usr/bin/` or `/bin`. It's more intuitive.

Reply:
the convention is that all tectonic operator binaries are built under the image's root folder.